test/test_conversion_driver_esm1p5.py

```diff
@@ -135,30 +163,37 @@ def test_convert_fields_file_list_fail_excepted(mock_process_with_exception):
     fake_file_path = Path("fake_file")
 
     _, failed = esm1p5_convert.convert_fields_file_list(
-        [fake_file_path], "fake_nc_write_dir")
+        [fake_file_path], "fake_nc_write_dir", delete_ff=True)
 
     assert failed[0][0] == fake_file_path
 
+    # Assert that no files removed
+    mock_os_remove.assert_not_called()
+
     # TODO: Testing the exception part of the reported failures will be easier
     # once um2nc specific exceptions are added.
 
 
-def test_convert_fields_file_list_fail_critical(mock_process_with_exception):
+def test_convert_fields_file_list_fail_critical(mock_process_with_exception, mock_os_remove):
     # Test that critical exceptions which are not allowed by ALLOWED_UM2NC_EXCEPTION_MESSAGES
     # are raised, and hence lead to the conversion crashing.
     generic_error_message = "Test error"
     mock_process_with_exception(generic_error_message)
     with pytest.raises(Exception) as exc_info:
         esm1p5_convert.convert_fields_file_list(
-            ["fake_file"], "fake_nc_write_dir")
+            ["fake_file"], "fake_nc_write_dir", delete_ff=True)
```

Comment:
The test failed here and the one 12 lines down too.

Reply:
These ones should be fixed now too